### DISCUSSION TITLE
Creating default permissions for all new repos within an org

### DISCUSSION THREAD

**Discussion by thaiknguyen** [community/community#68264] [New to GitHub]
I would like to set up an org where members of specific Teams are given default access to all repos (existing and any new repos created).  I do not believe there's such a way to do this, and have created a script using the APIs to iterate through all existing repos and add the Team as Collaborators.. this works fine for existing repos, but I would like to ensure anyone creating new repos would also have the same permissions granted...  namely, I have a Read-Only team, an Admin team, and a Write team, that I would like to be defaulted for the repos.

Other than having to run the script every so often, is there a better way to do this and set these repos up with the permissions by default?  I do not believe a template repo fits this need, as it does not look like it would inherit the permissions.

**Comment by queenofcorgis** [comment]
Hi there @thaiknguyen - I'm looking into this to see if/what's possible for you - sit tight 😄

**Reply by queenofcorgis** [reply]
This is not something that is possible - but I've passed your ask along to our team that works on org and repo roles.

**Comment by thaiknguyen** [comment]
Thanks.  Please let me now if the team has any suggestions, and also can you submit this as a feature request.

**Reply by queenofcorgis** [reply]
What you're doing with the API is probably the best work around - but yes when I passed this along to the team I did so as product feedback and added the appropriate label here.
